geo/src/lib.rs

```diff
@@ -150,13 +150,14 @@
 //! ## Miscellaneous
 //!
 //! - **[`Centroid`](Centroid)**: Calculate the centroid of a geometry
+//! - **[`ChaikinSmoothing`](ChaikinSmoothing)**: Smoothen `LineString`, `Polygon`, `MultiLineString` and `MultiPolygon` using Chaikins algorithm.
```

Comment:
Should be `Chaikin's`

Reply:
Okay, but this wasn't my typo! ;)

Reply:
Modify a line --> fix all conceivable problems with the line. I don't make the rules (I do).